server/package-lock.json

```diff
@@ -0,0 +1,757 @@
+{
```

Comment:
I don't think we need to commit this, we can add it to `.gitignore`

Reply:
This is needed (indicates what dependencies are needed)